fastly/base_fastly_service.go

```diff
@@ -178,7 +178,7 @@ func resourceDelete(serviceDef ServiceDefinition) schema.DeleteContextFunc {
 // resourceImport satisfies the Terraform resource schema Importer "interface"
 func resourceImport() *schema.ResourceImporter {
 	return &schema.ResourceImporter{
-		StateContext: func(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
unused arguments should be replaced with `_` to avoid warnings about it being unused.